src/control/server/mgmt_svc.go

```diff
@@ -530,110 +530,129 @@ func (svc *mgmtSvc) PoolDeleteACL(ctx context.Context, req *mgmtpb.DeleteACLReq)
 func (svc *mgmtSvc) BioHealthQuery(ctx context.Context, req *mgmtpb.BioHealthReq) (*mgmtpb.BioHealthResp, error) {
```

Comment:
With the updated logic, I think some of these storage methods could benefit from a little more explanation in the comment. Took a moment of study to figure out that this is just trying instances until one succeeds (meaning the instance had access to the requested device).

Reply:
Good call, I'll add it.